<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TitleWidget</class>
 <widget class="QWidget" name="TitleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>544</width>
    <height>56</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>544</width>
    <height>56</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>544</width>
    <height>56</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>544</width>
     <height>56</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>544</width>
     <height>56</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>544</width>
     <height>56</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel{background:transparent}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="imgs.qrc">:/img/green.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="title">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>16</y>
     <width>144</width>
     <height>26</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>144</width>
     <height>26</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>144</width>
     <height>26</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="imgs.qrc">:/img/douban_fm_text.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>24</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QToolButton" name="volumeButton">
   <property name="geometry">
    <rect>
     <x>496</x>
     <y>20</y>
     <width>24</width>
     <height>19</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>24</width>
     <height>19</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>24</width>
     <height>19</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton{border-image: url(:/img/volume_normal.png);}
QToolButton:hover{border-image: url(:/img/volume_clicked.png);}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="volume">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>15</y>
     <width>71</width>
     <height>29</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="userLogin">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>35</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>文泉驿微米黑</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>未登录</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToolButton" name="minimizeButton">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>24</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="imgs.qrc"/>
 </resources>
 <connections/>
</ui>
